### ISSUE TITLE
Accordion Light Theme: Link or button text lacks 4.5:1 contrast ratio on hover or focus

### ISSUE THREAD

**Issue by scurker** [dequelabs/cauldron#902]
Original Auditor Issue: https://axeauditor.dequecloud.com/test-run/f6620faa-6d1e-11ec-a0b7-97f3ccf70959/issue/7a43c47e-adf8-11ed-9477-4f5a3a9c2076?sortField=identifier&sortDir=asc&page=0&activeTab=dt-issue&pageSize=50&viewMode=issues&row=0&issuesCount%5B0%5D=&issuesCount%5B1%5D=&filter%5Bgroup_id%5D%5B0%5D=42b8f3a2-ac6e-11ed-adb1-3bfa602e0928&filter%5Bgroup_id%5D%5B1%5D=df2b20c0-1317-11ed-acb1-c7cbc98656eb

## Description
Light Theme: Link or button text lacks 4.5:1 contrast ratio on hover or focus

### Element Description:
-Accordion 1
-Accordion 2
-Accordion 3
Location of the element: Below the Try, it out - "Standard" and "Controlled" heading

Link color code (On hover): #3C7AAE
Background color code: #F0F2F5
Contrast ratio: 4.1:1

### Steps to reproduce:
Expand the above-mentioned accordion 
Hover over the above-mentioned element, the button lacks color contrast on hover

### Description of the issue:
The contrast ratio between the link text and its background is not at least 4.5:1 on hover.

### Recommendation to fix
RULE : 
Small text (under 18 point regular font or 14 point bold font) MUST have a contrast ratio of at least 4.5 to 1 with the background.

#### HOW TO FIX: 
Fix this issue by adjusting the color of the hovered or focused link or button text and/or background to increase the contrast to at least 4.5 to 1.

#### REFERENCE: 
Deque University: https://dequeuniversity.com/class/visual-design2/contrast/text-against-background

## BACKGROUND: 
People who have low vision or are colorblind may have difficulty reading text if the contrast between the text its background is insufficient. When the contrast ratio between text and its background is adequate, people who have low vision or are colorblind are more likely to be able to read the text.

![accordion contrast issue on hover/focus](https://user-images.githubusercontent.com/1062039/219794739-f29cd592-11ea-4497-9e24-0e33342c416f.png)

**Comment by scurker** [comment]
Fix is just removing the gray background when expanded (see ux pin): https://preview.uxpin.com/520e80315786a67ba43a3ca50598e6d69782627c#/pages/161871161/simulate/no-panels?mode=i

**Comment by scurker** [comment]
The updated design doesn't 100% match what's currently in uxpin, we're just addressing the specific accessibility issue here and sync the design up with #856
